constants.go

```diff
@@ -429,6 +429,8 @@ const (
 	NetIQ = "netiq"
 	// ADFS is Microsoft Active Directory Federation Services
 	ADFS = "adfs"
+	// Ping is an identity provider.
+	Ping = "ping"
```

Comment:
What's the official name here?

Reply:
My understanding is that the underlying platform is PingIdentity, PingOne is the cloud/SaaS hosted product, and PingFederate is the on-prem product. This workaround seems to be required in all cases. Perhaps renaming it to `PingIdentity` would be more specific?

Reply:
I've clarified the name in the constant's comment but can still rename it if desired.

Reply:
Okay sounds good to me.